spec/raven/integrations/rack_spec.rb

```diff
@@ -102,7 +86,7 @@
     end
 
     stack = Raven::Rack.new(Rack::Lint.new(app))
-    expect { stack.call(env) }.to_not raise_error(Rack::Lint::LintError)
```

Comment:
Yep - latest version of RSpec doesn't allow you to specify an error class when expecting a block `to_not` raise an error.